fineract-core/src/main/java/org/apache/fineract/portfolio/savings/domain/SavingsHelper.java

```diff
@@ -161,6 +161,8 @@ private LocalDate determineInterestPostingPeriodEndDateFrom(final LocalDate peri
                 }
                 periodEndDate = periodEndDate.with(TemporalAdjusters.lastDayOfMonth());
             break;
+            case ACTIVATIONDATE:
```

Comment:
Can you please explain the math here?

Reply:
@adamsaghy Yes this is going to increment based on the activation day and increments monthly wrt date. As far as naming convention, please guide me here this is my first time around here at Fineract so please feel free to let me know what other changes I need to be doing here.

Reply:
My major concern is the story and the implementation is not matching... Please check the dates in the story and the dates in your test cases.

Reply:
@DheerajKumar19 Have you had the chance to check my concern regarding the implementation and the story requirement?